sphinx/themes/basic/static/searchtools.js_t

```diff
@@ -256,7 +256,7 @@ var Search = {
             displayNextItem();
           });
         } else if (DOCUMENTATION_OPTIONS.HAS_SOURCE) {
-          $.ajax({url: DOCUMENTATION_OPTIONS.URL_ROOT + '_sources/' + item[0] + '.txt',
+          $.ajax({url: DOCUMENTATION_OPTIONS.URL_ROOT + '_sources/' + item[5] + '.txt',
```

Comment:
Thanks for the hints, but I'm kinda lost.
How can I test this JavaScript search stuff locally?